hc/api/tests/test_create_check.py

```diff
@@ -16,7 +16,7 @@ def post(self, data, expected_error=None):
 
         if expected_error:
             self.assertEqual(r.status_code, 400)
-            ### Assert that the expected error is the response error
+             ### Assert that the expected error is the response error
```

Comment:
Check if the response message returned is in the list of expected errors. Seems like `expected_error ` is defined somewhere in the code.